popups.py

```diff
@@ -519,6 +520,31 @@ def diag_failed_popup():
     window.close()
     return
 
+def bad_exclusions_popup():
+    '''
+    Show information on exclusions that are not recommended by Cisco.
+    '''
+
+    bad_exclusions_list_text = "\n".join([str(x) for x in bad_exclusions_list])
+    layout = [
+        [sg.Text("THIS IS NOT A LIST OF EXCLUSIONS IN YOUR ENVIRONMENT, BUT A LIST OF EXCLUSIONS WE RECOMMEND AGAINST!")],
+        [sg.Text("For the most up to date list and additional information refer to")],
+        [sg.Text("https://www.cisco.com/c/en/us/support/docs/security/amp-endpoints/213681-best-practices-for-amp-for-endpoint-excl.html")],

```

Comment:
This isn't clickable, can't even be highlighted

Reply:
Added hyperlink.  Added popups for authentication errors before exiting the program.